src/components/AssetBreadcrumb/stories/AssetBreadcrumb.stories.tsx

```diff
@@ -30,6 +31,15 @@ class CogniteClient extends MockCogniteClient {
   };
 }
 
+const retriewAssets = async (internalId: InternalId): Promise<Asset[]> => {
```

Comment:
Rename `internalId ` to `assetId` , to keep consistency

Reply:
It's not just rename, we shouldn't take object `{ id:  id }` as a parameter, but rather just `id: number`

Reply:
But `InternalId` type means `{id: number}`

Reply:
yeah,  that's what I am saying. why we take object here? we just need a number, hence 
`async (assetId: CogniteInternalId): Promise<Asset[]>` is a correct signature

Reply:
oh, I see and this makes sense